src/pages/StockDetails/StockDetails.jsx

```
@@ -1,7 +1,79 @@
+                <span>-1312323123.567 </span>
+                <span>(-0.213123%)</span>
+              </p>
+            </div>
+          </div>
+        </div>
+
+        {/* Bookmark Button */}
+        <div className="flex gap-2 items-center">
+          {/* Bookmark Button */}
+          <Button className="flex items-center gap-2">
+            {true ? (
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)